src/js/plugins/plugin.mobile_nav.js

```diff
@@ -70,9 +77,32 @@ BookReader.prototype.initToolbar = (function (super_) {
         }
       });
 
-      // High contrast button
-      $drawerEl.find('.high-contrast-button').click(
-        () => this.refs.$br.toggleClass('high-contrast'));
+
+      // Dinamically creates styles combining different filters for BookReaders imgs
+      // based on filters checkbox
+      $drawerEl.find('.BRcheckbox-filters').click(
+        () => {
+          const br = this;
+          $("#filtersStyle").remove();
+          let filterStr = "";
+          let filterWebkitStr = "";
+
+          $('.BRcheckbox-filters').each(
+            function () {
+              br.refs.$br.removeClass("filter-applied");
+              if($(this).is(':checked')){
+                br.refs.$br.addClass($(this).attr("filter-applied"));
+                filterWebkitStr = filterWebkitStr + FILTERLIST[$(this).attr("id")];
+                filterStr = filterStr + FILTERLIST[$(this).attr("id")];
+              }
+            }
+          )
+          const filtersSheet = document.createElement('style');
+          filtersSheet.id = "filtersStyle";
+          filtersSheet.innerHTML = ".BRtwopageview, .BRpageview img {filter:" + filterStr + "; -webkit-filter:" + filterWebkitStr + ";}";
```

Comment:
Also: Template strings :)
```suggestion
          filtersSheet.innerHTML = `.BRtwopageview, .BRpageview img {
              filter: ${filterStr};
              -webkit-filter: ${filterWebkitStr};
           }`;
```